src/engine/client/input.cpp

```diff
@@ -318,6 +347,18 @@ bool CInput::KeyState(int Key) const
 	return m_aInputState[Key>=KEY_MOUSE_1 ? Key : SDL_GetScancodeFromKey(KeyToKeycode(Key))];
 }
 
+void CInput::SetCompositionWindowPosition(float X, float Y, float H)
+{
+	// TODO: use HiDPI scale instead of config screen settings (requires #2827)
```

Comment:
@TsFreddie What to do here? `Rect.x = X / m_ScreenHiDPIScale` for x, y, h?

Reply:
yes. :)